bundles/org.openhab.binding.networkupstools/src/main/resources/OH-INF/i18n/networkupstools.properties

```
@@ -82,9 +82,9 @@ channel-type.networkupstools.ups-status.state.option.OL = Online
 channel-type.networkupstools.ups-status.state.option.OB = On battery
 channel-type.networkupstools.ups-status.state.option.LB = Low battery
 channel-type.networkupstools.ups-status.state.option.CHRG = Charging
-channel-type.networkupstools.ups-status.state.option.OL CHRG = Online, charging
-channel-type.networkupstools.ups-status.state.option.OL LB = Online, low battery
-channel-type.networkupstools.ups-status.state.option.OL CHRG LB = Online, charging, low battery
+channel-type.networkupstools.ups-status.state.option.OL_CHRG = Online, charging
+channel-type.networkupstools.ups-status.state.option.OL_LB = Online, low battery
+channel-type.networkupstools.ups-status.state.option.OL_CHRG_LB = Online, charging, low battery
```

This should fix the issue I would expect. You can test this by changing the text to something random, build the binding, and see if it shows the random text as option description:

```suggestion
channel-type.networkupstools.ups-status.state.option.OL\ CHRG = Online, charging
channel-type.networkupstools.ups-status.state.option.OL\ LB = Online, low battery
channel-type.networkupstools.ups-status.state.option.OL\ CHRG\ LB = Online, charging, low battery
```

---

Yes, backslash space actually works. 
My random text shows up.

@hilbrand is it worth to proceed here? Shall I extend the PR to the other affected plugins?

---

I think it would be a good idea if you can fix all. 
Just rename your PR in this case.